docs/index.md

```diff
@@ -20,7 +20,7 @@ An [Organization Owner](https://docs.couchbase.com/cloud/organizations/organizat
 This API key is then used for authenticating the Terraform Provider against Couchbase Capella.
 
 
-[!IMPORTANT]  
+**IMPORTANT**  
```

Comment:
The previous formatting was GH-flavoured markdown - or at least a variant thereof - and should have worked on the Registry, but apparently didn't.
This will at least look better, even if it fails to generate an admonition box.

In the longterm, perhaps the world will see sense, and abandon a melange of markdown in favour of asciidoc.  :crossed_fingers: